.gitignore

```diff
@@ -1 +1,15 @@
-/dist/
+dist
+cabal-dev
+*.o
+*.hi
+*.chi
+*.chs.h
+*.dyn_o
+*.dyn_hi
+.hpc
+.hsenv
+.cabal-sandbox/
+cabal.sandbox.config
+*.prof
+*.aux
+*.hp
```

Comment:
I'm not excited about catch all ignore files.  If you want this behavior you could still do it in [your global git ignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).